subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1170,7 +1170,8 @@ static void le_read_buffer_size(struct net_buf *buf, struct net_buf **evt)
 	rp->le_max_num = CONFIG_BT_BUF_ACL_TX_COUNT;
 }
 
-#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
+#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_SYNC_ISO) || \
```

Comment:
`defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_SYNC_ISO)` should be simplified to `defined(CONFIG_BT_CTLR_BROADCAST_ISO)`

Reply:
@wopu-ot The reason to add `CONFIG_BT_CTLR_SYNC_ISO` by this PR is because Zephyr Host seems to send the HCI LE Read Buffer Size v2 despite the command being conditionally excluded in HCI (the below is something that I only verified in spec today):

Bluetooth Core Specification Version 5.2 Vol 4, Part E Section 3 Table 3.1
>LE Read Buffer
Size command
4.0 The HCI_LE_Read_Buffer_Size
command returns the size of the
HCI buffers. These buffers are
used by the LE Controller to
buffer data that is to be transmitted.
E E [v1] M
[v2] C45

> C45: Mandatory if LE Feature (Connected Isochronous Stream - Master), or LE Feature
(Connected Isochronous Stream - Slave), or LE Feature (Isochronous
Broadcaster) is supported, otherwise excluded.

@jhedberg @Vudentz @joerchan will the host be updated to not use HCI LE Read Buffer Size v2 if an application is built as Synchronized Receiver only (example, this sample wants to use the HCI command: https://docs.zephyrproject.org/latest/samples/bluetooth/iso_receive/README.html)

Reply:
@Thalley Could you update the feature check in the host? Perhaps check for the command itself being supported instead of feature?

Reply:
@joerchan Sure, coming right up

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/36355

Reply:
Not that this does actually not fix the problem entirely, as the host does not (yet) have a configuration for sync receiver only (it's in the progress of being done, see https://github.com/zephyrproject-rtos/zephyr/pull/35493 for preliminary work)

Reply:
Do I understand correctly that #36355 makes this PR unnecessary?

Reply:
@wopu-ot For a Zephyr-only built, perhaps. But if someone wants to use Zephyr Controller with another host I guess this still makes sense.

Reply:
@wopu-ot and @Thalley as the spec excludes the HCI command for synchronized receiver, this PR is not needed.